org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeCoordinatorHandler.java

```diff
@@ -180,6 +181,31 @@ public void initialize() {
             initializeNetwork = true;
         }
 
+        // Process the network key
+        try {
+            logger.debug("Network Key String {}", networkKeyString);
+            networkKey = new ZigBeeKey(networkKeyString);
+        } catch (IllegalArgumentException e) {
```

Comment:
May I suggest that we add a logging statement in case of `IllegalArgumentException` as well? I think that this is useful in case that a user provides a custom network key via the bridge configuration, which is invalid.

I see that one can deduce that this was the case from the log (because in this case one will see the log message "Network key initialised" below) - but this is somewhat implicit imho.